models/orderHistory.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require("mongoose");
+const Joi = require("joi");
+
+const orderHistorySchema = new mongoose.Schema({
+  userId: { type: String },
+  productId: { type: String },
+  address: {
+    name: { type: String },
+    mobile: { type: Number },
+    address1: { type: String },
+    town: { type: String },
+    city: { type: String },
+    state: { type: String },
+    pincode: { type: Number },
+  },
+  totalAmount: {
+    type: Number,
+    default: 0,
+  },
+  deliveredOn: { type: Number },
```

Comment:
Shouldn't this be date?

Reply:
in my mind, I am going to save epoch. That is why I took it as a type number.

Reply:
> I am going to save epoch. That is why I took it as a type number.

there is a high chance that you are missing the time zone information with epoch number. You should use https://en.wikipedia.org/wiki/ISO_8601 time stamps